config/tools-manual.json

```diff
@@ -29,15 +273,88 @@
                     "repoUrl": "https://gitlab.com/djencks/asyncapi-asciidoc-template"
                 },
                 "filters": {
-                    "language": "javascript",
+                    "language": "JavaScript",
                     "technology": [
                         "react"
                     ],
                     "categories": [
-                        "documentation generator"
+                        "documentation-generator"
                     ],
-                    "hasCommercial": false,
-                    "isAsyncAPIOwner": false
+                    "hasCommercial": false
+                }
+            },
+            {
+                "title": "Springwolf",
+                "description": "Automated documentation for async APIs built with Spring Boot. Like Springfox for AsyncAPI. Auto-generates an AsyncAPI document and a web UI.",
+                "links": {
+                    "websiteUrl": "https://springwolf.github.io/",
+                    "repoUrl": "https://github.com/springwolf/springwolf-core"
+                },
+                "filters": {
+                    "language": "Java",
+                    "technology": ["Springboot"],
+                    "categories": ["code-first", "documentation-generator"]
+                }
+            },
+            {
+                "title": "KnstEventBus",
+                "description": "AsyncApi code-first tools for c#. Generates document and view.",
+                "links": {
+                    "repoUrl": "https://github.com/d0972058277/KnstEventBus"
+                },
+                "filters": {
+                    "language": "C#",
+                    "technology": ["ASP.NET", ".NET"],
+                    "categories": ["code-first", "documentation-generator"]
+                }
+            },
+            {
+                "title": "AsyncAPI Generator",
+                "description": "Generator is a tool that you can use to generate whatever you want basing on the AsyncAPI specification file as an input.",
+                "links": {
+                    "docsUrl": "https://www.asyncapi.com/docs/tools/generator",
+                    "repoUrl": "https://github.com/asyncapi/generator"
+                },
+                "filters": {
+                    "language": "JavaScript",
+                    "technology": ["Node js", "Markdown"],
+                    "categories": ["code-generator", "documentation-generator"]
+                }
+            },
+            {
+                "title": "Bump.sh",
+                "description": "OpenAPI 2 & 3 / AsyncAPI 2 documentation generator, with automatic changelog and visual diff.",
+                "links": {
+                    "websiteUrl": "https://bump.sh/",
+                    "repoUrl": "https://github.com/bump-sh"
```

Comment:
this is not repo URL

Reply:
Yeah, it's not a repo URL, but still it is the organization where users might want to see the codebase available open source. In this way, they will get that the collective repositories and projects are handled in this organization and can further raise issues/discussions regarding their choice.

Reply:
Done with the changes :+1: